### ISSUE TITLE
Can't start boot2docker under Windows 7.

### ISSUE THREAD

**Issue by hantsy** [boot2docker/windows-installer#95]
I am using the following stack.
1. Boot2Docker 1.5(only installed Boot2Docker, and Management Tools)
2. VirtualBox 4.3.26 which I had installed before.
3. Msysgit (git version 1.9.5.msysgit.1) which I also installed before.

After I installed Boot2Docker, and tried to run the app icon, the cmd windows quit silently.

I read some issues about the problem, and tried to init the boot2docker vm manually.
I had removed the boot2docker-vm in VirtualBox , and also removed the network settings.
1. run `boot2docker -v init`.
   
   <pre>
   E:\Users\hantsy>boot2docker -v init
   Boot2Docker-cli version: v1.5.0
   Git commit: ccd9032
   2015/04/05 12:30:02 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    showvminfo boot2docker-vm --machinereadable
   2015/04/05 12:30:02 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    showvminfo boot2docker-vm --machinereadable
   2015/04/05 12:30:02 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    list vms
   2015/04/05 12:30:02 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    createvm --name boot2docker-vm --register
   Virtual machine 'boot2docker-vm' is created and registered.
   UUID: e1905172-8c1b-4a64-9aa5-b061f5bbd0e3
   Settings file: 'E:\Users\hantsy\VirtualBox VMs\boot2docker-vm\boot2docker-vm.vbo
   x'
   2015/04/05 12:30:03 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    showvminfo boot2docker-vm --machinereadable
   2015/04/05 12:30:03 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    setextradata boot2docker-vm VBoxInternal/CPUM/EnableHVP 1
   2015/04/05 12:30:03 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    modifyvm boot2docker-vm --firmware bios --bioslogofadein off --bioslogofadeout
   off --bioslogodisplaytime 0 --biosbootmenu disabled --natdnshostresolver1 off --
   natdnsproxy1 off --ostype Linux26_64 --cpus 4 --memory 2048 --vram 8 --acpi on -
   -ioapic on --rtcuseutc on --cpuhotplug off --pae on --longmode on --synthcpu off
    --hpet on --hwvirtex on --triplefaultreset off --nestedpaging on --largepages o
   n --vtxvpid on --vtxux off --accelerate3d off --uart1 0x3F8 4 --uartmode1 server
    \\.\pipe\boot2docker-vm --boot1 dvd
   2015/04/05 12:30:03 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    showvminfo boot2docker-vm --machinereadable
   2015/04/05 12:30:03 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    modifyvm boot2docker-vm --nic1 nat --nictype1 virtio --cableconnected1 on
   2015/04/05 12:30:03 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    controlvm boot2docker-vm natpf1 ssh,tcp,127.0.0.1,2022,,22
   2015/04/05 12:30:03 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    list hostonlyifs
   2015/04/05 12:30:04 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    list dhcpservers
   2015/04/05 12:30:04 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    modifyvm boot2docker-vm --nic2 hostonly --nictype2 virtio --cableconnected2 on
   --hostonlyadapter2 VirtualBox Host-Only Ethernet Adapter
   2015/04/05 12:30:04 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    storagectl boot2docker-vm --name SATA --add sata --portcount 4 --hostiocache on
    --bootable on
   2015/04/05 12:30:04 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    storageattach boot2docker-vm --storagectl SATA --port 0 --device 0 --type dvddr
   ive --medium E:\Users\hantsy\.boot2docker\boot2docker.iso
   Converting from raw image file="stdin" to file="E:\Users\hantsy\VirtualBox VMs\b
   oot2docker-vm\boot2docker-vm.vmdk"...
   Creating dynamic image with size 20971520000 bytes (20000MB)...
   Initializing disk with ssh keys
   WRITING: boot2docker, please format-me
                                0000000 0000000 0000000 00000000035 00000000000 014
   423
                             ustar 00
                     0000000 0000000
   
                                            boot2docker, please format-me
   
   
   
   
   
                                                                            .ssh
   
                0000700 0000000 0000000 00000000000 00000000000 007767  5
   
             ustar 00
     0000000 0000000
   
                            .ssh/authorized_keys
                                                0000644 0000000 0000000 00000000621
    00000000000 013133
                                             ustar 00
                                     0000000 0000000
   
                                                            ssh-rsa AAAAB3NzaC1yc2E
   AAAADAQABAAABAQCfib7mhY3E54agwQsAVHI4d3inGd9re9WvzyK9w24FecNzqX9/ETfbrw+oEcEL4zT
   bGyHME1+AmOvdt4ZzmVQfAcZGS3Bta2BoNtrhEk2goFSG+jTdhXIZ8kkV9vpml8f8Sqv+PzI5cKI6j5Q
   NwAIEJ/Y/xpo000w48rwL5s55NgLP1JB+XhDPtZqW6ZD+1SZcOopfiOfXYIFkKUDCn9n67L9EcHplcaf
   K4Qgu0LSberAuf9GmkyPN2D3VckgAr4cHtXpGxNEMNB+BacU2uV5b8fQRZphLqUQJquhOaXNBTAcQZg6
   h+vg130sCkAGqP64wHfq7nLw7QSHTQ0onrYNX hantsy@HANTSY-T540P
   
                                  .ssh/authorized_keys2
                                                      0000644 0000000 0000000 00000
   000621 00000000000 013215
                                                   ustar 00
                                           0000000 0000000
   
                                                                  ssh-rsa AAAAB3Nza
   C1yc2EAAAADAQABAAABAQCfib7mhY3E54agwQsAVHI4d3inGd9re9WvzyK9w24FecNzqX9/ETfbrw+oE
   cEL4zTbGyHME1+AmOvdt4ZzmVQfAcZGS3Bta2BoNtrhEk2goFSG+jTdhXIZ8kkV9vpml8f8Sqv+PzI5c
   KI6j5QNwAIEJ/Y/xpo000w48rwL5s55NgLP1JB+XhDPtZqW6ZD+1SZcOopfiOfXYIFkKUDCn9n67L9Ec
   HplcafK4Qgu0LSberAuf9GmkyPN2D3VckgAr4cHtXpGxNEMNB+BacU2uV5b8fQRZphLqUQJquhOaXNBT
   AcQZg6h+vg130sCkAGqP64wHfq7nLw7QSHTQ0onrYNX hantsy@HANTSY-T540P
   
   
   
   
   
   
   
   
   
   
   
   
   
   
   
   -----
   2015/04/05 12:30:16 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    storageattach boot2docker-vm --storagectl SATA --port 1 --device 0 --type hdd -
   -medium E:\Users\hantsy\VirtualBox VMs\boot2docker-vm\boot2docker-vm.vmdk
   </pre>
2. run `boot2docker -v up`.
   
   <pre>
   E:\Users\hantsy>boot2docker -v up
   Boot2Docker-cli version: v1.5.0
   Git commit: ccd9032
   2015/04/05 12:31:31 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    showvminfo boot2docker-vm --machinereadable
   2015/04/05 12:31:31 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    guestproperty set boot2docker-vm /VirtualBox/GuestAdd/SharedFolders/MountPrefix
    /
   2015/04/05 12:31:31 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    guestproperty set boot2docker-vm /VirtualBox/GuestAdd/SharedFolders/MountDir /
   2015/04/05 12:31:31 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    sharedfolder add boot2docker-vm --name c/Users --hostpath C:\Users --automount
   2015/04/05 12:31:31 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    setextradata boot2docker-vm VBoxInternal2/SharedFoldersEnableSymlinksCreate/c/U
   sers 1
   2015/04/05 12:31:31 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    startvm boot2docker-vm --type headless
   Waiting for VM "boot2docker-vm" to power on...
   VM "boot2docker-vm" has been successfully started.
   2015/04/05 12:31:37 executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
    showvminfo boot2docker-vm --machinereadable
   Waiting for VM and Docker daemon to start...
   .Connecting to tcp://localhost:2022 (attempt #0).Connecting to tcp://localhost:2
   022 (attempt #0).Connecting to tcp://localhost:2022 (attempt #0).Connecting to t
   cp://localhost:2022 (attempt #0).Connecting to tcp://localhost:2022 (attempt #0)
   .Connecting to tcp://localhost:2022 (attempt #0).Connecting to tcp://localhost:2
   022 (attempt #0).Connecting to tcp://localhost:2022 (attempt #0).Connecting to t
   cp://localhost:2022 (attempt #0).Connecting to tcp://localhost:2022 (attempt #0)
   .Connecting to tcp://localhost:2022 (attempt #0).Connecting to tcp://localhost:2
   022 (attempt #0).Connecting to tcp://localhost:2022 (attempt #0).Connecting to t
   cp://localhost:2022 (attempt #0).Connecting to tcp://localhost:2022 (attempt #0)
   .Connecting to tcp://localhost:2022 (attempt #0).Connecting to tcp://localhost:2
   022 (attempt #0).Connecting to tcp://localhost:2022 (attempt #0).Connecting to t
   cp://localhost:2022 (attempt #0).Connecting to tcp://localhost:2022 (attempt #0)
   .Connecting to tcp://localhost:2022 (attempt #0).Connecting to tcp://localhost:2
   022 (attempt #0).Connecting to tcp://localhost:2022 (attempt #0)2015/04/05 12:31
   :59 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o IdentitiesOnly=yes
   -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=quiet -p
    2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost ip addr show dev e
   th1
   SSH returned: 4: eth1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fa
   st state UP qlen 1000
       link/ether 08:00:27:ac:c6:3b brd ff:ff:ff:ff:ff:ff
       inet 192.168.59.104/24 brd 192.168.59.255 scope global eth1
          valid_lft forever preferred_lft forever
       inet6 fe80::a00:27ff:feac:c63b/64 scope link
          valid_lft forever preferred_lft forever
   
   END SSH
   VM Host-only IP address: 192.168.59.104
   Waiting for Docker daemon to start...
   o2015/04/05 12:32:01 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   cat: can't open '/var/run/docker.pid': No such file or directory
   Error requesting socket: exit status 1
   o2015/04/05 12:32:02 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   cat: can't open '/var/run/docker.pid': No such file or directory
   Error requesting socket: exit status 1
   o2015/04/05 12:32:03 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   cat: can't open '/var/run/docker.pid': No such file or directory
   Error requesting socket: exit status 1
   o2015/04/05 12:32:04 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   cat: can't open '/var/run/docker.pid': No such file or directory
   Error requesting socket: exit status 1
   o2015/04/05 12:32:05 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   cat: can't open '/var/run/docker.pid': No such file or directory
   Error requesting socket: exit status 1
   o2015/04/05 12:32:05 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   cat: can't open '/var/run/docker.pid': No such file or directory
   Error requesting socket: exit status 1
   o2015/04/05 12:32:06 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   cat: can't open '/var/run/docker.pid': No such file or directory
   Error requesting socket: exit status 1
   o2015/04/05 12:32:07 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   cat: can't open '/var/run/docker.pid': No such file or directory
   Error requesting socket: exit status 1
   o2015/04/05 12:32:07 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   cat: can't open '/var/run/docker.pid': No such file or directory
   Error requesting socket: exit status 1
   o2015/04/05 12:32:08 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   cat: can't open '/var/run/docker.pid': No such file or directory
   Error requesting socket: exit status 1
   o2015/04/05 12:32:09 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   cat: can't open '/var/run/docker.pid': No such file or directory
   Error requesting socket: exit status 1
   o2015/04/05 12:32:10 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   cat: can't open '/var/run/docker.pid': No such file or directory
   Error requesting socket: exit status 1
   o2015/04/05 12:32:10 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   cat: can't open '/var/run/docker.pid': No such file or directory
   Error requesting socket: exit status 1
   o2015/04/05 12:32:11 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   Error requesting socket: exit status 255
   o2015/04/05 12:32:13 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   Error requesting socket: exit status 255
   o2015/04/05 12:32:15 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   Error requesting socket: exit status 255
   o2015/04/05 12:32:16 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Id
   entitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o
   LogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost g
   rep tcp:// /proc/$(cat /var/run/docker.pid)/cmdline
   SSH returned: tcp://0.0.0.0:2376
   
   END SSH
   2015/04/05 12:32:16 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Ide
   ntitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o L
   ogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost ip
    addr show dev eth1
   SSH returned: 4: eth1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fa
   st state UP qlen 1000
       link/ether 08:00:27:ac:c6:3b brd ff:ff:ff:ff:ff:ff
       inet 192.168.59.104/24 brd 192.168.59.255 scope global eth1
          valid_lft forever preferred_lft forever
       inet6 fe80::a00:27ff:feac:c63b/64 scope link
          valid_lft forever preferred_lft forever
   
   END SSH
   
   Started.
   2015/04/05 12:32:17 executing: C:\Program Files (x86)\Git\bin\ssh.exe ssh -o Ide
   ntitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o L
   ogLevel=quiet -p 2022 -i E:\Users\hantsy\.ssh\id_boot2docker docker@localhost ta
   r c /home/docker/.docker/*.pem
   tar: Removing leading `/' from member names
   Writing E:\Users\hantsy\.boot2docker\certs\boot2docker-vm\ca.pem
   Writing E:\Users\hantsy\.boot2docker\certs\boot2docker-vm\cert.pem
   Writing E:\Users\hantsy\.boot2docker\certs\boot2docker-vm\key.pem
   Docker client does not run on Windows for now. Please use
       "boot2docker" ssh
   to SSH into the VM instead.
   </pre>
3. run `boot2docker ssh`
   
   <pre>
   E:\Users\hantsy>boot2docker ssh
                           ##        .
                     ## ## ##       ==
                  ## ## ## ##      ===
              /""""""""""""""""\___/ ===
         ~~~ {~~ ~~~~ ~~~ ~~~~ ~~ ~ /  ===- ~~~
              \______ o          __/
                \    \        __/
                 \____\______/
    _                 _   ____     _            _
   | |__   ___   ___ | |_|___ \ __| | ___   ___| | _____ _ __
   | '_ \ / _ \ / _ \| __| __) / _` |/ _ \ / __| |/ / _ \ '__|
   | |_) | (_) | (_) | |_ / __/ (_| | (_) | (__|   <  __/ |
   |_.__/ \___/ \___/ \__|_____\__,_|\___/ \___|_|\_\___|_|
   Boot2Docker version 1.5.0, build master : a66bce5 - Tue Feb 10 23:31:27 UTC 2015
   
   Docker version 1.5.0, build a8a31ef
   docker@boot2docker:~$ lsConnection to localhost closed by remote host.
   error in run: exit status 65535
   
   </pre>

In the second step, the error is the Docker client does not work.
in the 3rd step, when entered Boot2Docker ssh, it quit soon.

Any help here? THanks.

**Comment by hantsy** [comment]
I have to uninstall my existed VirutalBox(4.3.26) and install the shipped one(4.3.20)...it worked.

Fixed.

**Comment by tianon** [comment]
Sorry, this repository is long-since deprecated in favor of [Docker Toolbox](https://github.com/docker/toolbox) (whose usage is now also discouraged in favor of [Docker for Windows](https://www.docker.com/docker-windows) and [Docker for Mac](https://www.docker.com/docker-mac)).
